<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCreate</class>
 <widget class="QDialog" name="DialogCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>720</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>New Account Creation</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DialogCreate
{

}</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>491</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Copperplate Gothic Bold&quot;;</string>
   </property>
   <property name="title">
    <string>NEW ACCOUNT CREATION</string>
   </property>
   <widget class="QLineEdit" name="password1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#password1
{
	font: 12pt &quot;Andalus&quot;;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="mail">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Andalus&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surname">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Andalus&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>GENDER :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>EMAIL :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="countType">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choisir le type de compte à créer&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#countType
{
font: 12pt &quot;Andalus&quot;;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Current account</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Savings account</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>FIRSTNAME :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>CONFIRM PASSWORD :</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateB">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#date
{
font: 12pt &quot;Andalus&quot;;
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="gender">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#gender
{
font: 12pt &quot;Andalus&quot;;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Woman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Man</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Child</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No binary</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>COUNTRY :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>PHONE NUMBER :</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="saveData">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Copperplate Gothic Bold'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Validate this compte creation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#commandLinkButton
{
	background-color: rgb(211, 255, 212);
	font: 12pt &quot;Jokerman&quot;;
}</string>
    </property>
    <property name="text">
     <string>Create New Account (^_^)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>SURNAME :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>430</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#password
{
	font: 12pt &quot;Andalus&quot;;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>ACCOUNT TYPE :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>PASSWORD :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="country">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pays d'origine&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#country
{
	font: 12pt &quot;Andalus&quot;;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Austria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Belgium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Benin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Burkina-Faso</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Canada</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>France</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Egypt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Germany</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ghana</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Japon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lybie</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Niger</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nigeria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Senegal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>South Africa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Togo</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>BIRTH DATE :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstname">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Andalus&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Andalus&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#cancel
{	
	font: 8.5pt &quot;Batang&quot;;
}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/MesImages/MesImages/Screenshot_20210211-234707.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 127);
font: 75 11pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>No more worry for Funds</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;CityBlueprint&quot;;
color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>BLUE LIGHTED BANK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogCreate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>688</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>720</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
